src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  display: flex;
+  flex-direction: column ;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background-color: $white;
+  border: 1px solid $borderColor;
+  border-radius: 5px;
+
```

Comment:
If I'm not mistaken, you can do card without flex. All elements will be in a column. Display should be set between elements, which need it.

Reply:
Check @Yuliia-kucheriavenko comment, it's redundant to use `flex` for the card, each block element will be displayed on the new line by default